pkg/skuba/actions/cluster/init/assets.go

```diff
@@ -111,4 +112,59 @@ ignore-volume-az=true
 No special operation needs to be performed if you deployed the whole
 infrastructure using the terraform state files provided by SUSE.
 `, "~", "`")
+
+	vSphereReadme = strings.ReplaceAll(`# ~vSphere~ integration
+
+Create a file inside this directory named ~vsphere.conf~, with the [supported contents](https://github.com/kubernetes/cloud-provider-vsphere/blob/master/docs/book/cloud_config.md):
+
+You can find a template named vsphere.conf.template inside this directory.
+
+~~~
+[Global]
+user = "<VC_USERNAME>"
+password = "<VC_PASSWORD>"
+port = "443"
+insecure-flag = "1"
+[VirtualCenter <VC_IP_OR_FQDN>]
+datacenters = "<VC_DATACENTERS>"
+[Workspace]
+server = "<VC_IP_OR_FQDN>"
+datacenter = "<VC_DATACENTER>"
+default-datastore = "<VC_DATASTORE>"
+resourcepool-path = "<VC_RESOURCEPOOL_PATH>"
+folder = "<VC_VM_FOLDER>"
+[Disk]
+scsicontrollertype = pvscsi
+[Network]
+public-network = "VM Network"
+~~~
+
+If this file exists the cloud integration for ~vSphere~ will be automatically
+enabled when you bootstrap the cluster.
+
+## Important
+
+When the cloud provider integration is enabled, it's very important to bootstrap
+and join nodes with the node names same as ~vSphere~ virtual machine's hostnames,
+as these names will be used by services to reconcile node metadata.
+`, "~", "`")
+
+	vSphereCloudConfTemplate = `[Global]
+user = "<VC_USERNAME>"
+password = "<VC_PASSWORD>"
+port = "443"
+insecure-flag = "1"
+[VirtualCenter <VC_IP_OR_FQDN>]
+datacenters = "<VC_DATACENTERS>"
+[Workspace]
+server = "<VC_IP_OR_FQDN>"
+datacenter = "<VC_DATACENTER>"
+default-datastore = "<VC_DATASTORE>"
+resourcepool-path = "<VC_RESOURCEPOOL_PATH>"
+folder = "<VC_VM_FOLDER>"
+[Disk]
+scsicontrollertype = pvscsi
+[Network]
+public-network = "VM Network"
+`
```

Comment:
Should we add below to the template?
```
[Labels]
region = "<VC_DATACENTER_TAG>"
zone = "<VC_CLUSTER_TAG>"
```

Reply:
Thought about it. To me, I prefer to keep things at minimal in template. Zones and regions requires extra setup on vCenter and not given a correct value will break the deployment. So it's been leaved out by purpose to ensure user had went through the document and this is something  they are intentionally going for. Again, that is just my 2 cents.